### DISCUSSION TITLE
[Feature] Rematch faces of individual pictures

### DISCUSSION THREAD

**Discussion by localweights** [immich-app/immich#4012] [Feature Request]
### The feature

I have a few face groups where immich has put several faces/things under the same group. E.g according to Immich apparently my Golden Retriever looks like my daughter

I’d like to correct this by removing/re tagging the face of individual photos. 

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of  #2472

**Reply by localweights** [reply]
Cool, I read that linked item as a group of pictures wrongly tagged, not an individual picture within a group
